drivers/flash/flash_stm32.c

```diff
@@ -224,13 +224,7 @@ static const struct flash_driver_api flash_stm32_api = {
 #ifdef CONFIG_FLASH_PAGE_LAYOUT
 	.page_layout = flash_stm32_page_layout,
 #endif
-#if defined(CONFIG_SOC_SERIES_STM32F0X)
```

Comment:
Should this be #if defined(FLASH_WRITE_BLOCK_SIZE) instead?

Reply:
Yes it should !